tests/test_misc.py

```diff
@@ -2309,7 +2309,7 @@ def test_emergencyrecover(node_factory, bitcoind):
     """
     Test emergencyrecover
     """
-    l1, l2 = node_factory.get_nodes(2, opts=[{}, {}])
+    l1, l2 = node_factory.get_nodes(2, opts=[{'allow_broken_log': True, 'may_reconnect': True}, {}])
```

Comment:
both nodes need `'may_reconnect': True`

```suggestion
    l1, l2 = node_factory.get_nodes(2, opts=[{'allow_broken_log': True, 'may_reconnect': True}, {'may_reconnect': True}])
```

Reply:
Done